From 227b2f07c42a6f7b51957fb2fba61cde16f348ae Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Tue, 19 Aug 2014 22:03:04 -0400
Subject: [crypto] x509: Need to export x509_request_asymmetric_key()

Message-id: <1408485785-777-33-git-send-email-vgoyal@redhat.com>
Patchwork-id: 88386
O-Subject: [RHEL7.1 PATCH 32/33] X.509: Need to export x509_request_asymmetric_key()
Bugzilla: 1129779
RH-Acked-by: David Howells <dhowells@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

BZ:1129779

commit cf5b50fd2d70fdd907d433bcebaf8d89a9490334
Author: David Howells <dhowells@redhat.com>
Date: Sun, 3 Aug 2014 12:54:48 +0100

Need to export x509_request_asymmetric_key() so that PKCS#7 can use it if
compiled as a module.

Reported-by: James Morris <jmorris@namei.org>
Signed-off-by: David Howells <dhowells@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/crypto/asymmetric_keys/x509_public_key.c b/crypto/asymmetric_keys/x509_public_key.c
index 18c3890..11c561b 100644
--- a/crypto/asymmetric_keys/x509_public_key.c
+++ b/crypto/asymmetric_keys/x509_public_key.c
@@ -96,6 +96,7 @@ struct key *x509_request_asymmetric_key(struct key *keyring,
    key_serial(key_ref_to_ptr(key)));
  return key_ref_to_ptr(key);
 }
+EXPORT_SYMBOL_GPL(x509_request_asymmetric_key);
 
 /*
  * Set up the signature parameters in an X.509 certificate.  This involves
-- 
1.7.1